frontend/src/components/taskSelection/taskList.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { FormattedMessage, FormattedRelative } from 'react-intl';
+import ReactPlaceholder from 'react-placeholder';
+import { useQueryParam, NumberParam } from 'use-query-params';
+
+import messages from './messages';
+import { useFetch } from '../../hooks/UseFetch';
+import { compareTaskId } from '../../utils/sorting';
+import { colours } from './map';
+import { LockIcon } from '../svgIcons';
+import { PaginatorLine, howManyPages } from '../paginator'
+
+function TaskStatus({status}: Object) {
+  const dotSize = ['READY', 'LOCKED_FOR_MAPPING'].includes(status) ? '7px': '10px';
+  return (
+    <span>
+      <span className={`${['READY', 'LOCKED_FOR_MAPPING'].includes(status) && 'ba b--grey-light'} dib`}
+        style={{height: dotSize, width: dotSize, borderWidth: "2px", backgroundColor: colours[status]}} >
+      </span>
+      {status.startsWith('LOCKED_FOR_') &&
+        <LockIcon style={{width: "12px", height: "12px", paddingTop: "1px"}} className="v-mid pl1"/>
+      }
+      <span className="pl2">
+        <FormattedMessage {...messages[`taskStatus_${status}`]} />
+      </span>
+    </span>
+  );
+}
+
+function TaskItem({data, projectId}: Object) {
+  const activeTasks = useSelector(state => state.tasks.get('activeTasks'));
+  const activeProject = useSelector(state => state.tasks.get('activeProject'));
+  const dispatch = useDispatch();
+
+  function selectTask() {
+    if (activeTasks.includes(data.taskId)) {
+      dispatch({ type: 'SET_ACTIVE_TASKS', activeTasks: activeTasks.filter(i => i !== data.taskId) });
+    } else {
+      dispatch({ type: 'SET_ACTIVE_TASKS', activeTasks: [data.taskId] });
+    }
+    if (activeProject !== projectId) {
+      dispatch({ type: 'SET_ACTIVE_PROJECT', activeProject: projectId });
+    }
+  }
+
+  return (
+    <div className={`cf db pv3 ba br1 mt2 ${activeTasks.includes(data.taskId) ? 'b--blue-dark bw1' : 'b--tan bw1'}`}>
```

Comment:
These left margins get a bit weird when you make the page smaller.
<img width="347" alt="image" src="https://user-images.githubusercontent.com/283343/66114422-d1db9b80-e59c-11e9-9af1-ddbcb4caf6d8.png">

Reply:
I'll solve it on a future PR.